src/optional.h

```diff
@@ -5,22 +5,16 @@
 #ifndef BITCOIN_OPTIONAL_H
 #define BITCOIN_OPTIONAL_H
 
+#include <optional>
 #include <utility>
 
-#include <boost/optional.hpp>
-
 //! Substitute for C++17 std::optional
+//! DEPRECATED use std::optional in new code.
```

Comment:
To avoid conflicts with other pulls or backports? If so it could mention an acceptable time to remove this alias.

Reply:
I'd say when 0.21.1 is out or when 0.22 is about to get branched off, whichever happens first.

Reply:
Sounds good to me.